libafl/src/monitors/mod.rs

```diff
@@ -502,6 +502,10 @@ impl ClientStats {
 /// The monitor trait keeps track of all the client's monitor, and offers methods to display them.
 pub trait Monitor {
     /// The client monitor (mutable)
+    ///
+    /// This method is for internal usage only, you shall never call this method directly.
```

Comment:
Does it make sense to keep this in the trait, then? We could also provide some `std_` functions that monitors can choose to use and remove the functions from the trait completely

Reply:
I don't quite understand what you said about std. The real `client_stats` data is maintained by individual monitors, so they have to implement this trait method to expose this. Normal functions can not access this without trait generics, i think 🤔

Reply:
I mean, why have a function in the trait that nobody is supposed to access anyway(?)

Reply:
The default implementation of other trait methods of `Monitor` use this. You could check that out by going to `Monitor`'s definition. For `CombinedMonitor`, we manually implement those methods to avoid invoking this.

Reply:
Yes my point is, maybe they shouldn't use this. Instead, we could have a common method they _can_ call when they want to, and remove this method from the trait

Reply:
It appears to me that it may not be very easy to design such individual function. Instead, we may further decouple the entire client_stats to an independent entity, and Monitors are just consumers to this entity. This separation could solve the problem, and lead to more flexibility. However, this requires even larger rewriting, so if you think it's ok, I'll try it when I have time :)

Reply:
Yes we can merge this first and then fix it later